sys/shell/shell.c

```diff
@@ -104,6 +121,10 @@ static shell_command_handler_t find_handler(
         handler = search_commands(_builtin_cmds, command);
     }
 
+    if (handler == NULL) {
```

Comment:
So if there is an `xfa` version of the command and a "classic" build-in version of the command, the classic version will be preferred? This should at the very least be documented.

Reply:
hmya. same with built-in and user specified. Is that documented? If not, that's a doc buf for a followup.

Reply:
No it's not, but this is literally just a note you can stick to the doc `SHELL_COMMAND`. The doc for build-ins, yes, that can be done as a follow-up.

Reply:
done

Reply:
Can you provide a link? I can't find it.

Reply:
I stuck a note to `SHELL_COMMAND()`:
https://github.com/kaspar030/RIOT/blob/a36f9ce7744edba9fa473a9a9d7a5aab2953c827/sys/include/shell.h#L206-L209